mantis/cli/utils.py

```diff
@@ -381,9 +381,11 @@ def process_single_position_v2(
             func,
             input_dataset,
             output_path / Path(*input_data_path.parts[-3:]),
-            input_channel_indices=None,
+            None,
```

Comment:
probably more readable if we keep the argument's name (i.e `input_chanel_indices=None`)